File: server/models/eventModel.js
import mongoose from "mongoose";

const Schema = mongoose.Schema;

const eventSchema = new Schema({
  name: {
    type: String,
    required: true,
  },
  start: {
    type: String,
    required: true,
  },
  end: {
    type: String,
    required: true,
  },
  description: {
    type: String,
    required: false,
  },
});

const Event = mongoose.model("Event", eventSchema);
export default Event;

File: server/server/backend/models/eventModel.py
from django.db import models
from .calendarModel import Calendar


class Event(models.Model):
    name = models.CharField(max_length=30)
    description = models.TextField()
    start = models.TextField()
    end = models.TextField()

    calendar = models.ForeignKey(Calendar, on_delete=models.CASCADE)
